lib/webtask.js

```diff
@@ -708,6 +748,48 @@ exports.ensure_authenticated = function(ctx) {
             });
         }
 
+        function validate_auth_token(auth) {
+            var auth_token;
+            var decoded_auth_token;
+            var userinfo;
+            var secret = new Buffer((auth.secret || DEFAULT_AUTH0_CLIENT_SECRET), 'base64');
+
+            try {
+                auth_token = req.query.key || (req.headers['authorization']
+                    .match(/\s.+$/)[0]
```

Comment:
```js
.match(/^bearer\s+(,+)$/i)[1]
```

Then, no need to do the replace and we are validating that a `Bearer` token has, in fact, been passed.